cfpb/capital-framework-sandbox
# Form examples

Update to the forms examples with our latest direction in layout and atomic changes.
## Additions
- Adds a filter form examples in both atomic and cfgov-refresh patterns to track changes that will be needed in cfgov-refresh.
## Changes
- Updates the form examples of labels and inputs
- Separates the layout from visual styles. 
## Testing
- check out latest `prerelease-ver-4` in capital framework repo
- in this repo run `npm run build && npm start`
- check out the atomic filtering form at the bottom of the page
## Review
- @Scotchester 
- @KimberlyMunoz 
- @mistergone 
- @cfarm 

[Preview this PR without the whitespace changes](?w=0)
## Screenshots

<img width="776" alt="screen shot 2016-09-21 at 2 52 04 pm" src="https://cloud.githubusercontent.com/assets/1280430/18726626/fa43bbb6-800a-11e6-8f82-09d3d7c7997e.png">
## Checklist
